src/node_http_parser.cc

```diff
@@ -707,25 +832,37 @@ class Parser : public AsyncWrap, public StreamListener {
   Local<Object> current_buffer_;
   size_t current_buffer_len_;
   char* current_buffer_data_;
+#ifdef NODE_EXPERIMENTAL_HTTP
+  unsigned int executing_;
```

Comment:
Can you name this e.g. `execute_depth_` or change it to bool if it's only meant to be at most 1?

Reply:
I'd rather do `execute_depth_`. `bool` is great, but `int` is better for tracking potential bugs later on.